README.md

```diff
@@ -1,22 +1,21 @@
-# quickstart-atlassian-jira
-## Jira Software and Jira Service Desk Data Center on the AWS Cloud
+# quickstart-atlassian-crowd
+## Crowd Software Data Center on the AWS Cloud
 
-Use this Quick Start to deploy one of the following Jira products from Atlassian on the AWS Cloud:
+Use this Quick Start to deploy Crowd Data Center on the AWS Cloud.
 
-* Jira Software Data Center
-* Jira Service Desk Data Center
+Crowd lets you centralize identity management. Manage users from multiple directories - Active Directory, LDAP, OpenLDAP or Microsoft Azure AD - and control application authentication permissions in one single location.
 
-This Quick Start uses the [Atlassian Standard Infrastructure (ASI)](https://fwd.aws/xYyYy) as a foundation. You can choose to build a new ASI for your deployment or deploy Jira into your existing ASI. You can also deploy Bitbucket Data Center and Confluence Data Center within the same ASI.
+This Quick Start uses the [Atlassian Standard Infrastructure (ASI)](https://fwd.aws/xYyYy) as a foundation. You can choose to build a new ASI for your deployment or deploy Crowd into your existing ASI. You can also deploy Bitbucket Data Center and Confluence Data Center within the same ASI.
 
-![Quick Start architecture for Jira on AWS](https://d1.awsstatic.com/partner-network/QuickStart/datasheets/jira-arch-on-aws.9d422797475ea5bd5d38f009ca8c540736f5449e.png)
+![Quick Start architecture for Crowd on AWS](https://d1.awsstatic.com/partner-network/QuickStart/datasheets/jira-arch-on-aws.9d422797475ea5bd5d38f009ca8c540736f5449e.png)
```

Comment:
Update Jira architecture diagram reference to a Crowd specific one, for this an architecture diagram needs to be created for Crowd. It should be similar to the [Jira architecture diagram](https://camo.githubusercontent.com/7f4004be546a8d5f9bd83510df9e9446ba977237/68747470733a2f2f64312e6177737374617469632e636f6d2f706172746e65722d6e6574776f726b2f517569636b53746172742f646174617368656574732f6a6972612d617263682d6f6e2d6177732e396434323237393734373565613562643564333866303039636138633534303733366635343439652e706e67) but refer to Crowd Nodes instead of Jira Nodes.